app/components/ui/molecules/PeoplePicker.js

```diff
@@ -0,0 +1,159 @@
+/* eslint-disable react/no-unused-prop-types */
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Button } from '@pubsweet/ui'
+import { th } from '@pubsweet/ui-toolkit'
+import { Box, Flex } from 'grid-styled'
+
+import SelectedItem from '../atoms/SelectedItem'
+import PersonPod from '../atoms/PersonPod'
+
+const SelectedItems = ({ selection, onCloseClick }) => (
+  <Flex>
+    {selection.map(person => (
+      <Box key={person.id} mr={1}>
+        <SelectedItem
+          label={person.name}
+          onCloseClick={() => onCloseClick(person)}
+        />
+      </Box>
+    ))}
+  </Flex>
+)
+
+const SuccessMessage = styled.div`
+  color: ${th('colorSuccess')};
+`
+
+const SelectionHint = ({ selection, minSelection, maxSelection }) => {
+  const selectionLength = selection.length
+  if (selectionLength < minSelection) {
+    const numRequired = minSelection - selectionLength
+    return (
+      <div>
+        {numRequired} more suggestion{numRequired === 1 ? '' : 's'} required
+      </div>
+    )
+  }
+  if (selectionLength >= maxSelection) {
+    return <SuccessMessage>Maximum {maxSelection} choices</SuccessMessage>
+  }
+  return null
```

Comment:
What do you think of having the `PeoplePickerBody` being the one responsible for showing/hiding this message, rather than `SelectionHint` returning an element in some situations and returning nothing in other situations?

I feel like I should be able to look at `PeoplePickerBody`, without knowing anything about how `SelectionHint` works, and understand what kind of form it will take. As it is, I would expect _something_ to be rendered where it says `SelectionHint`.

To me, this is one of the benefits of thinking about building from an Atomic Design approach - composing from the ground up means that each component makes sense in total isolation.